packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/OvirtDetailsSection.tsx

```diff
@@ -31,7 +47,12 @@ export const OvirtDetailsSection: React.FC<DetailsSectionProps> = ({ data }) =>
 
       <DetailsItem title={''} content={''} />
 
-      <NameAndUiLinkDetailsItem resource={provider} />
+      <NameAndUiLinkDetailsItem
+        resource={provider}
+        canPatch={permissions.canPatch}
+        webUILinkText={t(`Red Hat Virtualization Manager UI`)}
+        calcWebUILink={getProviderUiContent}
```

Comment:
```suggestion
        webUILink={getProviderUiContent(provider)}
```

so at this point, we know the provider and we know how to calculate the link, my concern here is that the NameAndUiLinkDetailsItem is getting a function when it can get a regular string

Reply:
In current implementation there is no difference between using a function or using the calculated string as a prop.
So changing it to string is really better.